src/Enum/Role.php

```diff
@@ -23,8 +23,8 @@
 /**
  * @psalm-immutable
  */
-class Role extends Enum
+enum Role: String
```

Comment:
```suggestion
enum Role: string
```

From perspective: Less confusing 😄 .